docs/rst/user_guide/atmosphere/heterogeneous.rst

```diff
@@ -0,0 +1,194 @@
+.. _sec-atmosphere-heterogeneous:
```

Comment:
Same as for the homogeneous atmosphere guide: Since this document demonstrates usage, I think it's best to make a tutorial out of it (some of the code won't work btw, you'd have noticed that with a proper script).